msg/obstacle_distance.msg

```diff
@@ -1,6 +1,10 @@
 # Obstacle distances in front of the sensor.
 uint64 timestamp		# time since system start (microseconds)
 
+uint8 frame		#Coordinate frame of reference for the yaw rotation and offset of the sensor data. Defaults to MAV_FRAME_GLOBAL, which is North aligned. For body-mounted sensors use MAV_FRAME_BODY_FRD, which is vehicle front aligned.
+uint8 MAV_FRAME_GLOBAL = 0
```

Comment:
This should be MAV_FRAME_LOCAL_NED.

Reply:
See https://github.com/mavlink/mavlink/pull/1214

Agreed we should support LOCAL_NED too, although in this case it will be exactly the same as GLOBAL because we just care about the yaw offset.